src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -175,6 +175,14 @@ def _get_resource_type(self):
             resource_type = command_type.settings.get('resource_type', None) if command_type else None
         return resource_type
 
+    def _merge_kwargs(self, kwargs, base_kwargs=None):
+        base = base_kwargs if base_kwargs is not None else getattr(self, 'command_kwargs')
+        return _merge_kwargs(base, kwargs)
```

Comment:
Can we keep the argument order of (base_kwargs, patch_kwargs) consistent across each `_merge_kwargs` definition?
Less confusion and would prevent accidental switching of the two.

Reply:
Ironically, this caused other bugs because I didn't switch it in all spots.  But it's all fixed now!